TG-Netflix/src/pages/modal/ModalPage.jsx

```diff
@@ -1,11 +1,15 @@
 import Modal from '../../components/Modal/Modal';
 import Footer from '../../components/footer/footer';
+import movieDetailsMock from '../../mock-data/movie_details_similar.mock.json'
 
-export default function ModalPage() {
+// props worden uiteindelijk doorgegeven met Params of elders
+export default function ModalPage(movie) {
+  movie = movieDetailsMock
```

Comment:
Je importeert hier direct de mock JSON. Dus tussen de haakjes kan je movie weglaten want deze komt niet binnen vanuit een parent, toch? (kijk maar wat er gebeurd als je hem delete.)

je movie = movieDetailsMock mist een const 
het moet zijn const movie = movieDetailsMock of ander const movies (meervoud) als je movie niet weghaald als parameter in ModalPage. 

Als je bovenstaande aanpast wordt de movie prop op correcte wijze doorgegeven naar Modal.jsx

Reply:
Dit heb ik gedaan vanuit de verwachting dat er normaalgesproken wel props worden doorgegeven. Maar de connectie tussen de lanes en de modal is nog niet uitgewerkt, dus kunnen de props nog niet daadwerkelijk doorgegeven worden, en heb ik daarom `movie = movieDetailsMock` erin gezet, Dit overschrijft de prop waarde. Omdat de prop undefined is vanwege dat de modal op dit moment helemaal onafhankelijk staat van de movie components, moest ik hier een import plaatsen zodat de modal wel toegang heeft tot de mockdata.